integration_tests/e2e/refer/new/confirmOasys.cy.ts

```diff
@@ -9,6 +9,7 @@ import {
   referralFactory,
 } from '../../../../server/testutils/factories'
 import { OrganisationUtils } from '../../../../server/utils'
+import auth from '../../../mockApis/auth'
```

Comment:
Is it worth adding one test to the general Refer integration tests file (`integration_tests/e2e/refer.cy.ts`) to check an example of this auth restriction, similar to the test about roles?